packages/overlays/src/OverlayMixin.js

```diff
@@ -15,52 +16,110 @@ export const OverlayMixin = dedupeMixin(
             type: Boolean,
             reflect: true,
           },
-          popperConfig: Object,
+          config: {
+            type: Object,
+          },
+          closeEventName: {
+            type: String,
+          },
         };
       }
 
+      constructor() {
+        super();
+        this.config = {};
+        this.closeEventName = 'overlay-close';
+      }
+
       get opened() {
```

Comment:
having own setter/getter and `reflect: true` does not work well together 😅 

own setter/getter means you take care of initial attribute read and reflection yourself 😅 
I would suggest syncing the opened (we already have events for that) and not having a separate setter/getter which get's the data directly

Reply:
Anything that I changed in this PR that is relevant to your comment? I think it was like this before I touched it 😅